src/test/java/com/fasterxml/jackson/databind/deser/std/EmptyStringAsSingleValueTest.java

```diff
@@ -0,0 +1,115 @@
+package com.fasterxml.jackson.databind.deser.std;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.cfg.CoercionAction;
+import com.fasterxml.jackson.databind.cfg.CoercionInputShape;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+public class EmptyStringAsSingleValueTest {
+    private static ObjectMapper normalMapper() {
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
+        return mapper;
+    }
+
+    private static ObjectMapper coercionMapper() {
+        ObjectMapper mapper = normalMapper();
+        // same as XmlMapper
+        mapper.coercionConfigDefaults()
```

Comment:
Similar thing here, wrt coercion config defaults from `CoerceToBooleanTest` something like:

```
    private final ObjectMapper MAPPER_INT_TO_EMPTY = jsonMapperBuilder()
            .withCoercionConfig(LogicalType.Boolean, cfg ->
                cfg.setCoercion(CoercionInputShape.Integer, CoercionAction.AsEmpty))
            .build();

```

(modified to settings here of course)